src/meshlink.c

```diff
@@ -654,18 +512,8 @@ int check_port(meshlink_handle_t *mesh) {
 		int port = 0x1000 + (rand() & 0x7fff);
 
 		if(try_bind(port)) {
```

Comment:
In `try_bind()` there is a memory leak at `getaddrinfo()` which uses `ai = ai->ai_next` to traverse through addresses and never stored the allocated pointer address.

Reply:
This was fixed by Gurram's pull request, which I'll merge to master separately.